src/opnsense/mvc/app/models/OPNsense/Interfaces/VxLan.xml

```diff
@@ -0,0 +1,35 @@
+<model>
+    <mount>//OPNsense/Interfaces</mount>
+    <description>
+        the OPNsense "GridExample" application
```

Comment:
Oops

Reply:
hehe, I'll fix this one, wanted to test my own example howto, a bit too enthusiastic  :)